config/clickhouse/dist_config.xml

```diff
@@ -0,0 +1,27 @@
+<yandex>
+    <max_server_memory_usage_to_ram_ratio from_env="MAX_MEMORY_USAGE_RATIO" />
+
+    <remote_servers>
+        <cluster_one_sh>
+            <shard>
+                <replica>
+                    <host>localhost</host>
```

Comment:
I don't think this is right. The Clickhouse details should match with https://github.com/getsentry/snuba/blob/41d7fe76aaf8a594e8f6e84015607dcde3f67ad4/docker-compose.yml#L18